src/gui/torrentcategorydialog.cpp

```diff
@@ -77,6 +77,13 @@ QString TorrentCategoryDialog::createCategory(QWidget *parent, const QString &pa
                         , tr("Category with the given name already exists.\n"
                              "Please choose a different name and try again."));
         }
+        else if (newCategoryName.isEmpty())
```

Comment:
Wouldn't it be better to just disable "OK" button when category name is empty?

Reply:
Yup implemented that using a custom `slot` method and `connect`. Qt is fun!